Library/Formula/py2cairo.rb

```diff
@@ -30,7 +30,7 @@ def install
     # disable waf's python extension mode because it explicitly links libpython
     # https://code.google.com/p/waf/issues/detail?id=1531
     inreplace "src/wscript", "pyext", ""
-    ENV["LINKFLAGS"] = "-undefined dynamic_lookup"
+    ENV["LINKFLAGS"] = "-undefined dynamic_lookup" if OS.mac?
     ENV.append_to_cflags `python-config --includes`
```

Comment:
You can probably remove these there lines (or make them conditional on `OS.mac?`) and then get rid of the `mv` below.